website/templates/project/project.mako

```diff
@@ -256,11 +256,11 @@
             <div class="panel-body" style="display:none">
                 <dl id="citationList" class="citation-list">
                     <dt>APA</dt>
-                        <dd class="citation-text" data-bind="text: apa"></dd>
+                        <dd class="citation-text" data-bind="html: apa"></dd>
```

Comment:
@chennan47 Yes, however this causes us to lose the formatting that CiteProc is able to provide, hence why we are investigating the security of this.